README.md

```diff
@@ -4,97 +4,164 @@
 
-```bash
-$ nvm use 8
+### Send release email
+
+This commands send an email to our QE notifying them that a particular version has been released.
+
+Send the following Slack message to Atomist:
+```
+send release email
+```
+
+You will need to provide a release version (e.g. 2.3.0.Final).
+
+### Update after the BOM release
+
+This command updates all the repositories that depend on our BOM release.
+Firstly, it updates the BOM version in all of the Snowdrop examples. It sets a parent version and a Spring Boot property in the examples `pom.xml` files.
+Secondly, it updates the BOM repository by appropriately bumping the development version. It calculates the next qualifier (`Beta`, `SP`, `Final`) based on the release version.
+
+Send the following Slack message to Atomist:
+```
+update for release
+```
+
+You will need to provide an upstream version of the release (e.g. 2.3.0.Final).
+
+## Configuration
+
+### Licenses generator
+
+This client uses [Licenses Generator](https://github.com/snowdrop/licenses-generator) java application to generate the licenses. The jar file of the generator and its different configuration files are located in an `etc` directory.
+
+### Launcher catalog
+
+Launcher catalog uses custom IDs and version names for all the runtimes. Our version IDs and name formats are configurable in a `launcher` section of a `config/default.json` file.
+
+### Email
+
+Release notification email is send to a specific QE person. An email and a name of a person is configurable in an `email` section of a `config/default.json` file.
+
+## Running the client locally
+
+### Configure Atomist
+
+The minimal required configuration is a `$HOME/.atomist/client.config.json` file with an API key and a workspace ID:
+```json
+{
+  "apiKey": "API_KEY",
+  "workspaceIds": [
+    "WORKSPACE_ID"
+  ]
+}
 ```
 
-## Miscellaneous
+You can get them from our [Atomist configuration dashboard](https://app.atomist.com).
 
-Run tests: `npm run test` (will need an env var named GITHUB_TOKEN to be present and contain the user's Github token)
+For all the Atomist prerequisites see its [documentation](https://docs.atomist.com/developer/prerequisites).
 
-Run a single test: `TEST=${TEST_FILE_NAME} npm run test:one`
+### Setup Node.js environment
 
-Lint code (and fix issues automatically): `npm run lint:fix`
+Node.js 8 or newer is required to run the client.
+We recommend using the [nvm](https://github.com/nvm-sh/nvm) tool for your Node.js version control.
 
-## Run as container
+Once the Node.js is available, install the project dependencies:
+```bash
+$ npm ci
+```
+
+### Start the client
+
+We need to compile the Typescript files and then start the application:
+```bash
+$ npm run compile && npm start
+```
+
+> Note. If you will use an email sending command, make sure a valid `SENDGRIDKEY` environment variable is available before starting the client.
```

Comment:
What value should be pass to that variable? If we have more documentation on that aspect, we should point to it.

Reply:
I've never used this command. @geoand do you remember where did you store that key?

Reply:
I've never use this command successfully, I always send the email manually, however, I have this environment variable defined on my machine.

Reply:
I assume it requires registering with sendgrid.com and getting an api key from them…

Reply:
I don't remember unfortunately. You should be able to just use any key however

Reply:
```suggestion
> Note. If you intend to use an email sending command, you will need to register with sendgrid.com, get an API key from them and provide it as a `SENDGRIDKEY` environment variable before starting the client.
```